.github/workflows/liquid.yml

```diff
@@ -1,39 +1,36 @@
 name: Liquid
 on: [push, pull_request]
+
+env:
+  BUNDLE_PATH: "vendor/bundle"
```

Comment:
Can this environment variable be removed? Since https://github.com/ruby/setup-ruby#bundle-config says

> To perform caching, this action will use bundle config --local path $PWD/vendor/bundle.
Therefore, the Bundler path should not be changed in your workflow for the cache to work (no bundle config path).